### PR TITLE
Make compatible with webpack

### PR DESCRIPTION
**Pull request by satya164**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,6 +7,7 @@
     "react-native-v1": "lib-rn/react-navigation.js",
     "web": "lib/react-navigation.web.js"
   },
+  "module": "lib/react-navigation.web.js",
   "webpack": "lib/react-navigation.web.js",
```
Comment (skevy) [inline package.json]: maybe take this out? Probably not needed.

File: website/config/webpack.config.dev.js [hunk 2]
```diff
@@ -80,7 +80,7 @@ module.exports = {
     alias: {
       // Support React Native Web
       // https://www.smashingmagazine.com/2016/08/a-glimpse-into-the-future-with-react-native-for-web/
-      'react-native': 'react-native-web',
+      // 'react-native': 'react-native-web',
```
Comment (skevy) [inline website/config/webpack.config.dev.js]: Remove the comment if we're not using it.

File: website/config/webpack.config.prod.js [hunk 3]
```diff
@@ -83,7 +83,7 @@ module.exports = {
     alias: {
       // Support React Native Web
       // https://www.smashingmagazine.com/2016/08/a-glimpse-into-the-future-with-react-native-for-web/
-      'react-native': 'react-native-web',
+      // 'react-native': 'react-native-web',
```
Comment (skevy) [inline website/config/webpack.config.prod.js]: ditto ^^

File: package.json (hunk 1 above)
Comment (satya164) [inline package.json]: @skevy out website doesn't work without it. probably difference between webpack 1 and 2

Comment (skevy) [inline package.json]: Hmm. ok.

**Pull request merged**